README.md

```diff
@@ -34,6 +34,7 @@ Data is parsed and stored as `x`:
     cat file.html | inspect html
     curl https://api.github.com/users/siadat/repos | inspect
 	curl https://api.github.com/repos/siadat/inspect/commits | inspect
+    inspect file.xml
```

Comment:
I think `parse_data` function is still [detecting](https://github.com/DanLindeman/inspect/blob/xml_support/lib/inspect.rb#L8) this as HTML, instead of XML. Do you think there is any way to distinguish the two? I can only think of using [strict](http://www.nokogiri.org/tutorials/ensuring_well_formed_markup.html) mode to raise a SyntaxError.